presto-client/src/main/java/io/prestosql/client/PrestoHeaders.java

```diff
@@ -50,5 +50,7 @@
     public static final String PRESTO_PAGE_NEXT_TOKEN = "X-Presto-Page-End-Sequence-Id";
     public static final String PRESTO_BUFFER_COMPLETE = "X-Presto-Buffer-Complete";
 
+    public static final String PRESTO_EXTRA_CREDENTIAL = "X-Presto-Extra-Credential";
```

Comment:
what is going to happen when client does not support this header and header is not empty?

Reply:
This is a header sent from clients. If they don't support it then they won't send it :)

Reply:
I would name it to `PRESTO_CLIENT_EXTRA_CREDENTIAL` then.